docs/strawberryrunners_subtitle.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Strawberry Runners Post-Processing
+tags:
+  - Strawberry Runners
+  - Subtitles
+  - VTT
+  - HOCR
+  - OCR
+  - Post-processing
+  - Background Processing
+---
+
+# Reviewing and adjusting the `subtitle` Post-Processor operations 
+
+As stated on the [Strawberry Runners overview page](docs/strawberryrunners.md), the `subtitle` action extracts textual values from subtitle/transcript VTT files and generates time/space transmuted OCR. This transmuted OCR can be used to search within a time-based video or audio file's corresponding subtitle/transcript VTT file(s), then navigate to the matching time of the video or audio file within a media viewer.
+
+We strongly recommend using caution when making any adjustments to the default `subtitle` configurations as this may result in unexpected issues with the transmuted OCR values in your Solr Index. Also, the `subtitle` Strawberry Runner Post-Processor needs to used with corresponding related default IIIF Configuration Form Settings. 
+
+# Pager Settings
```

Comment:
Why #pager?

Reply:
Oops, I used one of my other SBR guides as a base and didn't change this heading.